internal/querywatcher/query_watcher.go

```diff
@@ -53,23 +57,23 @@ var (
 	AdhocQueryChan chan AdhocQuery
 )
 
-// NewQueryWatcher initializes a new QueryWatcher.
-func NewQueryWatcher() *QueryWatcher {
+// NewQWatchManager initializes a new QWatchManager.
+func NewQWatchManager() *QWatchManager {
```

Comment:
I think a better name might be QueryManager. This component could be used to serve ad-hoc queries in the future (query execution without subscriptions) so tying the name to a particular command may not be a good idea.

Reply:
Done.